luaunit.lua

```diff
@@ -1986,8 +1986,8 @@ end
                     error( 'Instance must be a table or a function, not a '..type(instance)..', value '..prettystr(instance))
                 end
                 if M.LuaUnit.isClassMethod( name ) then
-                    className, methodName = M.LuaUnit.splitClassMethod( name )
-                    methodInstance = instance[methodName]
+                    local className, methodName = M.LuaUnit.splitClassMethod( name )
+                    local methodInstance = instance[methodName]
```

Comment:
For improved performance, it's usually advisable not to declare local variables within loops. Actually this applies to `name` and `instance` here, too. Ideally all four should be declared (`local name, instance, className, methodName`) _above_ the for-loop.